Content.Server/Doors/Systems/DoorSystem.cs

```diff
@@ -42,6 +42,7 @@ public override void Initialize()
         SubscribeLocalEvent<DoorComponent, WeldFinishedEvent>(OnWeldFinished);
         SubscribeLocalEvent<DoorComponent, WeldCancelledEvent>(OnWeldCancelled);
         SubscribeLocalEvent<DoorComponent, GotEmaggedEvent>(OnEmagged);
+        SubscribeLocalEvent<DoorComponent, DoorStateChangedEvent>(OnStateChanged);
```

Comment:
why does doorsystem need to check for this, isnt it the one setting this anyway? just do it when that happens

Reply:
SharedDoorSystem is raising the event and I need access to the server airlock component when finished.

Reply:
can do a virtual override

Reply:
Ok, I've overridden start opening and was able to check the last state and include bolting there. Thank you.

If I've unnecessarily copied anything please let me know.